workers/event_catcher/event_parser.rb

```diff
@@ -42,46 +42,42 @@ def self.parse_event(event)
     result[:username] = event.userName if event.userName.present?
 
     # Get the vm information
-    vm_key = "vm"          if event.props.key?("vm")
-    vm_key = "sourceVm"    if event.props.key?("sourceVm")
-    vm_key = "srcTemplate" if event.props.key?("srcTemplate")
+    vm_key = :vm          if event.props.key?(:vm)
+    vm_key = :sourceVm    if event.props.key?(:sourceVm)
+    vm_key = :srcTemplate if event.props.key?(:srcTemplate)
     if vm_key
-      vm_data = event.send(vm_key)
-
-      result[:vm_ems_ref]  = vm_data&.vm                 if vm_data&.vm
-      result[:vm_name]     = CGI.unescape(vm_data&.name) if vm_data&.name
-      result[:vm_location] = vm_data&.path               if vm_data&.path
-      result[:vm_uid_ems]  = vm_data&.uuid               if vm_data&.uuid
```

Comment:
These were properties added by the VimBroker and aren't present in the RbVmomi Event properties